File: app/src/main/java/com/example/modelapp/BaseApplication.java
package com.example.modelapp;

import android.app.Application;

import com.example.mvvm.config.Config;
import com.example.mvvm.config.Constant;

public class BaseApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Config.init(this)
                .add(Constant.API_HOST,"https://www.baidu.com/")
                .configure();
    }
}

File: app/src/main/java/com/example/modelapp/BaseApplication.kt
package com.example.modelapp

import android.app.Application
import com.baidu.mapapi.SDKInitializer

import com.example.mvvm.config.Config
import com.example.mvvm.config.Constant

class BaseApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        Config.init(this)
                .add(Constant.API_HOST, "https://www.baidu.com/")
                .configure()

        SDKInitializer.initialize(this)
    }
}
